### PR TITLE
Updated SIGs page

### LINKED ISSUE #3: SIG Pages
Each of the SIG pages has a list of projects, which overflows and doesn't really look organised.
- [ ] Use the `default` layout.
- [ ] Migrate all old projects (before the age of virtual expos) to a `<sig>_projects.json` file. 
- [ ] Load projects into the page in a year-wise division of tabs.
- [ ] Add links to virtual expo websites for virtual expo projects.

### PR DESCRIPTION
**Pull request by Shashankhs17**
Screenshot (102)
Screenshot (103)
Screenshot (104)
Screenshot (105)
Screenshot (106)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Site settings
 title: IEEE NITK
 description: IEEE NITK is a student branch of IEEE located at NITK Surathkal. This is the official website of IEEE NITK
-url: "https://ieee.nitk.ac.in" # the base hostname & protocol for your site e.g. http://willianjusten.com.br
+url: "" # the base hostname & protocol for your site e.g. http://willianjusten.com.br
```
Comment (gaurang2001) [inline _config.yml]: ahh don't change config please, even if you do locally, don't push it to remote

File: _layouts/default.html [hunk 2]
```diff
@@ -11,6 +11,8 @@
             });
         });
     </script>
+    <link rel="stylesheet" href="/assets/css/sig.css">
```
Comment (gaurang2001) [inline _layouts/default.html]: We don't want this stylesheet on every page. so link it on individual sigs HTML page - I think events uses a `js` and `css` parameter, follow that

**Review by gaurang2001 — CHANGES_REQUESTED** [review]
1. Display 2019 projects before 2018
2. Add WiE and SIGHT pages too - Affinity Groups

**Comment by Shashankhs17** [comment]
Screenshot (107)

Order of display corrected

Screenshot (108)
Screenshot (109)

Affinity groups added

File: _data/projects/diode.json [hunk 3]
```diff
@@ -1,113 +1,117 @@
...
-    {
-        "Title": "Undersampling and reconstruction of MR images",
-        "year":"",
-        "image":"res/img/placeh.jpg",
-        "Abstract": "Magnetic resonance imaging, one of the foremost non-invasive imaging modalities, suffers from a slow data acquisition rate.Patients must remain in the scanner for extended periods of time, which causes them considerable discomfort, and may even delay a critical diagnosis. Reduced scan times would not only address these issues but also decrease the cost, which is a critical factor in the Indian scenario.This project compares the results obtained by applying compressed sensing and sparse dictionary learning algorithms for MR rimage reconstruction. ",
-        "links": []
-    },
+    "2018" : [
+        {
+            "Title": "Boost Converter",
```
Comment (nishant-nayak) [inline _data/projects/diode.json:78]: @Shashankhs17 Are you sure the title for this should be Boost Converter? Looks like the first project in 2019 had the same title.

File: _layouts/default.html [hunk 4]
```diff
@@ -11,6 +11,7 @@
             });
         });
     </script>
+    <link rel="stylesheet" href="https://ieee.nitk.ac.in/virtual-expo/assets/css/style.css">
```
Comment (nishant-nayak) [inline _layouts/default.html]: @Shashankhs17 I assume you're calling this stylesheet only for the tab pills, if that's the case can you move just that CSS to this repository?

File: _data/projects/diode.json (hunk 3 above)
Comment (Shashankhs17) [reply _data/projects/diode.json:78]: Yeah, in the current website also it is like that only

File: _layouts/default.html (hunk 4 above)
Comment (Shashankhs17) [reply _layouts/default.html]: Yeah, it's called for the tab pills. I will move it to repository.

**Review by nishant-nayak — APPROVED** [review]
Looks good!

**Review by nishant-nayak — APPROVED** [review]

**Review by gaurang2001 — APPROVED** [review]

**Pull request merged**